.docksal/docksal.env

```diff
@@ -32,5 +32,3 @@ hostingenv="dev"
 MIGRATE_DB_SOURCE="pantheon_d7_project_machine_name"
 MIGRATE_DB_ENV="dev"
 
-# Sets the local development URL
-VIRTUAL_HOST=${hostingsite}.docksal.site
```

Comment:
We need this when working with contractors and clients to make sure they have the same URL.

Reply:
we need to assumed they wont be the same

Reply:
Why would it be different from folder and machine name?

If this is the hill you will die on, can we leave it commented out?